docs/maps/connect-to-ems.asciidoc

```diff
@@ -81,34 +81,53 @@ If you cannot connect to {ems} from the {kib} server or browser clients, and you
 
 {hosted-ems} is a self-managed version of {ems} offered as a Docker image that provides both the EMS basemaps and EMS boundaries. The image is bundled with basemaps up to zoom level 8. After connecting it to your {es} cluster for license validation, you have the option to download and configure a more detailed basemaps database.
 
-You can use +docker pull+ to download the {hosted-ems} image from the Elastic Docker registry.
-
+. Pull the {hosted-ems} Docker image.
++
 ifeval::["{release-state}"=="unreleased"]
-Version {version} of {hosted-ems} has not yet been released, so no Docker image is currently available for this version.
+WARNING: Version {version} of {hosted-ems} has not yet been released. 
+No Docker image is currently available for this version.
 endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
```

Comment:
Removing this `ifeval` conditional now displays the docker pull and start docker commands even though the image is not available yet. Is that intentional?

Reply:
Yes, this aligns the installation instructions with Kibana, from our current preview 

https://kibana_bk_195419.docs-preview.app.elstc.co/guide/en/kibana/master/docker.html